static/js/components/dashboard/courses/StatusMessages.js

```diff
@@ -314,10 +275,44 @@ export const calculateMessages = (props: CalculateMessagesProps) => {
     exams &&
     paid
   ) {
-    const message =
-      passedExam || failedExam
-        ? messageForAttemptedEdxExams(course, passedExam)
-        : messageForNotAttemptedEdxExam(course)
+    let message =
+      "There are currently no exams available. Please check back later."
+    let passedMsg = ""
+    if (passedExam || failedExam) {
+      passedMsg = passedExam
+        ? "You passed the exam. "
+        : "You did not pass the exam. "
+      if (course.has_to_pay && course.can_schedule_exam) {
+        passedMsg = `${passedMsg}If you want to re-take the exam, you need to pay again.`
```

Comment:
add that space here like `${passedMsg} If ...` that would save us from updating the test cases.